dist/js/main.js

```diff
@@ -170,21 +214,23 @@ function normalizeName(source) {
 
 function buildPopupContent(feature) {
     var subcontent = ''
-    var propertyNames = ['sector', 'state', 'scope_of_work', 'duration', 'bmgf_point', 'amount_us', 'grantee_organisation', 'beneficiary', 'title_of_grant', 'nature_of_work', 'focal_state', 'organisation']
+    var propertyNames = ['sector', 'state', 'scope_of_work', 'duration', 'bmgf_point', 'grantee_organisation', 'beneficiary', 'title_of_grant', 'nature_of_work', 'focal_state', 'organisation']
     for (var i = 0; i < propertyNames.length; i++) {
         subcontent = subcontent.concat('<p><strong>' + normalizeName(propertyNames[i]) + ': </strong>' + feature.properties[propertyNames[i]] + '</p>')
     }
-    return subcontent
+    return subcontent;
+
 }
 
 function getData(queryUrl) {
     $('.fa-spinner').addClass('fa-spin')
+    $('.fa-spinner').show()
     $.post(queryUrl, function (data) {
         $('.fa-spinner').removeClass('fa-spin')
         addDataToMap(data)
-    }).fail(function () {
-        console.log("error!")
-    });
+    }).done(function() {
+      $('.fa-spinner').hide()
```

Comment:
this can go into line 239. that block of code is executed when the post is done too